lib/inc/internal/nodes/config_node_simple_value.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "abstract_config_node.hpp"
+#include <internal/values/abstract_config_value.hpp>
+
+namespace hocon {
+
+    class config_node_simple_value : public abstract_config_node {
+    public:
+        config_node_simple_value(shared_token value);
+
+        shared_token get_token() const;
+        std::unique_ptr<abstract_config_value> const& get_value() const;
```

Comment:
Well some kind of pointer is necessary, because abstract type. You can't declare an abstract return type. How should this be handled?